src/test/scala/parsley/ExpressionParserTests.scala

```diff
@@ -25,7 +27,7 @@ class ExpressionParserTests extends ParsleyTest
     }
     it must "not leave the stack in an inconsistent state on failure" in
     {
-        val p = chainPost[Int]('1' #> 1, (col #>[Int => Int] (_ + 1)) <* '+')
+        val p = chainPost[Int]('1' #> 1, (col.#>[Int => Int](_ + 1)) <* '+')
```

Comment:
That type parameter no longer flies in an infix method, and Scala 2.13 infer it.  This syntax appeases all builds.